modules/fontrendering/include/modules/fontrendering/util/fontutils.h

```diff
@@ -57,12 +58,28 @@ getAvailableFonts(const std::string& fontPath = std::string());
  *
  * @return default path containing fonts shipped with Inviwo. Corresponds to the font directory
  *                 located in the font rendering module directory.
- *
  */
 std::string IVW_MODULE_FONTRENDERING_API getDefaultFontPath();
 
+/**
+ * \brief returns the default typeface of Inviwo for \p type along with the default font path and
+ * extension if \p fullPath is true
+ *
+ * @param type    requested font type
+ * @param path    if equal to FullPath::Yes, then the full path of the font including extension is
+ *                returned, only font name otherwise
+ * @return font name
+ */
+std::string IVW_MODULE_FONTRENDERING_API getFont(FontType type, FullPath path = FullPath::No);
```

Comment:
Is there are reason we are not returning a font?
Would make more sense instead of a path to a font.

Reply:
Font names (i.e. no path, no extension) are used in various properties, most often OptionProperties for font selection. Paths are used to refer to the actual font. And since Inviwo potentially supports multiple font directories... In addition, FreeType has support for otf, ttf, cff, and pcf fonts. Hence, tricky to figure out the extension.
What I would like to have is select a font family and then the style (bold, italics, semibold, etc.). But naming schemes are all over the place to do that. Some kind of heuristics needed.

Reply:
Font names can/could be retrieved from the font object if that is needed?